pkg/cmd/cli/cmd/set/triggers.go

```diff
@@ -512,6 +522,44 @@ func defaultNamespace(namespace, defaultNamespace string) string {
 	return namespace
 }
 
+// NewAnnotationTriggers creates a trigger definition from an object that can be triggered by the image
+// annotation.
+func NewAnnotationTriggers(obj runtime.Object) (*TriggerDefinition, error) {
+	m, err := meta.Accessor(obj)
+	if err != nil {
+		return nil, err
+	}
+
+	t := &TriggerDefinition{ConfigChange: true}
+	switch typed := obj.(type) {
+	case *kextensions.Deployment:
+		t.ConfigChange = !typed.Spec.Paused
```

Comment:
We don't override Paused though: https://github.com/openshift/origin/blob/e01d36c8b31b95459b66e71bb506e598a659b6d8/pkg/deploy/controller/generictrigger/controller.go#L45